core/src/main/kotlin/ch/ergon/dope/resolvable/expression/unaliased/type/conditional/IfNullExpression.kt

```diff
@@ -1,13 +1,14 @@
 package ch.ergon.dope.resolvable.expression.unaliased.type.conditional
 
 import ch.ergon.dope.resolvable.expression.UnaliasedExpression
+import ch.ergon.dope.resolvable.expression.unaliased.type.FunctionExpression
 import ch.ergon.dope.validtype.ValidType
 
 class IfNullExpression<T : ValidType>(
     firstExpression: UnaliasedExpression<T>,
     secondExpression: UnaliasedExpression<T>,
     vararg additionalExpressions: UnaliasedExpression<T>,
-) : ConditionalExpression<T>("IFNULL", firstExpression, secondExpression, *additionalExpressions)
+) : FunctionExpression<T>("IFNULL", firstExpression, secondExpression, *additionalExpressions)
```

Comment:
Aren't there a lot more function-expressions laying around? E.g. _decode_, _ltrim_, _ArrayLength_, ... Basically everything who inherits from FunctionOperator? Might be a good chance to do some more cleanup...

Reply:
There is definitely still potential for further refactoring. For the time being, I moved all of the function type expressions into a separate package.

Reply:
create ticket

Reply:
DOPE-257